demo/site/src/app/[visibility]/[domain]/[language]/news/api/route.ts

```diff
@@ -20,5 +20,12 @@ export async function GET(request: NextRequest, { params: { domain, language } }
         scope: { domain, language },
         ...result.data,
     });
-    return NextResponse.json(data);
+
+    // 7.5 min (CDN) + 7.5 min (Data Cache) = 15 min effective cache duration
+    return NextResponse.json(data, {
+        headers: {
+            "Cache-Control": "public, max-age=450, stale-while-revalidate=450",
```

Comment:
We recently increased the browser cache for DAM files (see https://github.com/vivid-planet/comet/pull/3926) and static assets (see https://github.com/vivid-planet/comet/pull/3924). Is this something we should consider for BFFs as well? Or do we treat data from the BFF the same way as pages/documents?